news/2018-09-12-new-featured-artist-kira.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "New Featured Artist: KIRA"
+date: 2018-09-12 15:00:00 +0000
+---
+
+Showcasing the best fusion of pop music and Vocaloids to date, we're proud to present our newest addition to the Featured Artist roster - **KIRA**!
+
+[![](https://assets.ppy.sh/artists/27/header.jpg)](https://osu.ppy.sh/beatmaps/artists/27)
+
+Inspired by osu! to start delving into the scene himself, **KIRA** has taken the Vocaloid music community by storm in recent years with several smash pop hits, such as *Circles* and *MONSTER*, the latter netting over *4.3million views on YouTube.* 
+
+Damn.
```

Comment:
wouldn't it be better to add it at the end of L11?

Reply:
i think it's fine